erpnext/regional/india/setup.py

```diff
@@ -629,15 +629,20 @@ def set_salary_components(docs):
 
 def set_tax_withholding_category(company):
 	accounts = []
+	fiscal_year = None
 	abbr = frappe.get_value("Company", company, "abbr")
 	tds_account = frappe.get_value("Account", 'TDS Payable - {0}'.format(abbr), 'name')
 
 	if company and tds_account:
 		accounts = [dict(company=company, account=tds_account)]
 
-	fiscal_year = get_fiscal_year(today(), company=company)[0]
-	docs = get_tds_details(accounts, fiscal_year)
+	try:
+		fiscal_year = get_fiscal_year(today(), verbose=0, company=company)[0]
+	except FiscalYearError:
+		pass
 
+	docs = get_tds_details(accounts, fiscal_year=None)
```

Comment:
It should be like this:
```suggestion
	fiscal_year = None
	try:
		fiscal_year = get_fiscal_year(today(), verbose=0, company=company)[0]
	except FiscalYearError:
		pass

	docs = get_tds_details(accounts, fiscal_year)
```

Reply:
I have declared fiscal_year = None above, removing it from the argument though, since it's unnecessary.